addons/product/tests/test_variants.py

```diff
@@ -643,6 +651,9 @@ def test_variant_images(self):
         # having changed the template image should not have changed these
         self.assertEqual(images[1:], self.variants.mapped('image_1920')[1:])
 
+        # last update changed for the variant without image
+        self.assertTrue(old_last_updates[0] < new_last_updates[0])
```

Comment:
According to the comment, it would be more natural if the code was using `variant_no_image['__last_update']` directly instead of comparing the first element of lists.

Similarly `old_last_updates` should not be a list if we are interested in a particular element.